lua/examples/example_of_dynamic_config_request.lua

```diff
@@ -40,21 +40,14 @@ dylib.add_grantor_entry_lb("198.51.100.0/25", addrs, dyc.gk)
 addrs[1] = { gt_ip = '203.0.113.4', gw_ip = '10.0.2.250' }
 dylib.update_grantor_entry_lb("198.51.100.0/25", addrs, dyc.gk)
 
--- Examples of temporarily changing global and block log levels.
-local old_log_level = staticlib.c.rte_log_get_global_level()
-staticlib.c.rte_log_set_global_level(staticlib.c.RTE_LOG_ERR)
+-- Examples of temporarily changing block log levels.
 
 local cpsc = staticlib.c.get_cps_conf()
 if cpsc == nil then
 	return "cps: failed to fetch config to update log level"
 end
 
-local old_cps_log_level = staticlib.c.rte_log_get_level(cpsc.log_type)
-if old_cps_log_level < 0 then
-	return "cps: failed to fetch log level"
-end
-
-ret = staticlib.c.rte_log_set_level(cpsc.log_type, staticlib.c.RTE_LOG_ERR)
+ret = staticlib.c.set_log_level_per_block("DYC", staticlib.c.RTE_LOG_ERR)
 if ret < 0 then
 	return "cps: failed to set new log level"
 end
```

Comment:
Why is this code which refers to the CPS block being changed in a patch for the dynamic configuration block? The same comment applies code below and my previous comment.